pycbc/frame/frame.py

```diff
@@ -191,6 +191,9 @@ def read_frame(location, channels, start_time=None,
     if sieve:
         logging.info("Using frames that match regexp: %s", sieve)
         lal.CacheSieve(cum_cache, 0, 0, None, None, sieve)
+    if start_time is not None and end_time is not None:
+        lal.CacheSieve(cum_cache, int(start_time), int(end_time),
```

Comment:
I think the end_time here should be a ceiling command.

Reply:
Yep, agreed

Reply:
`math.ceil` seems to work correctly on `lal.LIGOTimeGPS`. `numpy.ceil` does not.

Reply:
Pushed this change now.